packages/@lwc/style-compiler/src/__tests__/fixtures/aria-attributes/expected.js

```diff
@@ -2,5 +2,6 @@ function stylesheet(token, useActualHostSelector, useNativeDirPseudoclass) {
   var shadowSelector = token ? ("[" + token + "]") : "";
   var hostSelector = token ? ("[" + token + "-host]") : "";
   return ["[aria-labelledby]", shadowSelector, " {}[aria-labelledby=\"bar\"]", shadowSelector, " {}"].join('');
+  /*LWC compiler v2.9.0*/
```

Comment:
Something to keep in mind is that now post version upgrade all the tests will start failing. As part of the release process, post version update, all the test snapshots will have to be regenerated. Alternatively, we could transform output before saving the snapshot and strip out the actual version to make the snapshot stable cross-versions:

```suggestion
  /*LWC compiler vX.X.X*/
```

Reply:
I'm not sure how we could do the `vX.X.X` thing without modifying every snapshot test. I think the simplest is just to add `yarn test -u` to the `release:version` script.

Reply:
I am fine with this approach, I just want to raise the fact that the release commit will be harder to review. 

The alternative approach that I had in mind, would be to transform the output of the compilation [here](https://github.com/salesforce/lwc/blob/ee36ae984b46f1c16054e4ec24b035a5c3a304ad/packages/%40lwc/template-compiler/src/__tests__/fixtures.spec.ts#L29). We could reuse the regex from `@lwc/shared` to replace the version with `X.X.X`.

Either way, I am good.